features/storage/add_update_delete_volume.feature

```diff
@@ -68,19 +68,19 @@ Feature: Add, update remove volume to rc/dc and --overwrite option
     Then the step should succeed
     And I wait for the resource "pod" named "<%= pod.name %>" to disappear
     And a pod becomes ready with labels:
-      | app=mydb |
+      | deploymentconfig=mydb |
     When I execute on the pod:
       | grep | opt2 | /proc/mounts |
     Then the step should succeed
     When I run the :set_volume client command with:
       | resource | dc/mydb  |
       | action   | --remove |
       | name     | v1       |
-      | confirm  |          |
+      | confirm  | false    |
```

Comment:
I just not sure why the initial version adds confirm while it defaults to false,
--confirm=false: If true, confirm that you really want to remove multiple volumes

Reply:
@liangxia we can remove it.

Reply:
@liangxia ok, I just remove that line completely.